src/iocore/net/UnixNetVConnection.cc

```diff
@@ -1449,6 +1455,23 @@ UnixNetVConnection::remove_from_active_queue()
   }
 }
 
+void
+UnixNetVConnection::enable_inbound_connection_tracking(std::shared_ptr<ConnectionTracker::Group> group)
+{
+  ink_assert(nullptr == conn_track_group);
+  conn_track_group = std::move(group);
```

Comment:
The `move` is wrong. I'm not sure which way, but either you're copying to another shared pointer, in which case just assign, or it's a direct instance in which you now have dangling pointers elsewhere.

Reply:
I didn't originally write this with the move, but clang recommended it. It is a minor optimization and, if you think about it, it is correct. Remember that the move transfers the ownership from `group` to `conn_track_group` and leaves `group` empty (nullptr). When `group` goes out of scope later, nothing will happen. So this is fine. It avoids the reference count increment and then decrement when `group` later goes out of scope.